sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -460,13 +462,17 @@ public String getId() {
                 querySpec.setQueryText("select * from c where c.id = @id");
                 querySpec.setParameters(Collections.singletonList(new SqlParameter("@id",
                     UUID.randomUUID().toString())));
-                for (FeedRange feedRange : feedRanges) {
-                    CosmosQueryRequestOptions options = new CosmosQueryRequestOptions();
-                    options.setFeedRange(feedRange);
-                    CosmosPagedFlux<ObjectNode> cosmosPagedFlux = this.queryItems(querySpec, options,
-                        ObjectNode.class);
-                    fluxList.add(cosmosPagedFlux.byPage());
+
+                for (int i = 0; i < retryCount; i++) {
```

Comment:
We have a test case for this api , can you please quickly check that increasingly the value does touch more replicas, manual test would be sufficient.

Reply:
yes, manually test based on the test, it will touch more replicas.